crates/types/benches/topbid_ssz.rs

```diff
@@ -0,0 +1,51 @@
+use std::hint::black_box;
+
+use alloy_primitives::{Address, B256, U256};
+use criterion::{Criterion, criterion_group, criterion_main};
+use helix_types::BlsPublicKeyBytes;
+use ssz::Encode;
+use ssz_derive::{Decode, Encode};
+
+#[derive(Clone, Debug, Encode, Decode)]
+pub struct TopBidUpdate {
+    pub timestamp: u64,
+    pub slot: u64,
+    pub block_number: u64,
+    pub block_hash: B256,
+    pub parent_hash: B256,
+    pub builder_pubkey: BlsPublicKeyBytes,
+    pub fee_recipient: Address,
+    pub value: U256,
+}
+
+impl TopBidUpdate {
+    const SSZ_SIZE: usize = 188;
```

Comment:
can't you use `size_of::<TopBidUpdate>` here?

Reply:
wait - why is this defined in the bench? and not using the actual top bid update??